src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -5005,6 +5025,51 @@ export class ResultsViewPageStore extends AnalysisStore
         },
     });
 
+    readonly fusionMapperStores = remoteData<{
```

Comment:
this instantiates all the gene stores up front when it's possible that you really only need a store for the currently selected gene.  so investigate if it's possible to instantiate the stores lazily, when a gene tab is actually shown.